network-api/networkapi/templates/fragments/buyersguide/product_criterion_primary_info.html

```diff
@@ -15,7 +15,7 @@ <h3 class="tw-mb-8 tw-font-zilla tw-text-2xl tw-leading-7 tw-flex tw-items-cente
         </div>
     {% else %}
         <p class="rating pni-product-smaller-body mb-0">
-            {% trans value|extended_yes_no context value|extended_yes_no %}
+            {% trans value|extended_choice context value|extended_choice %}
```

Comment:
While not directly related to the field `ai_is_untrustworthy`, this line has been updated to use the updated template tag mentioned [here](https://github.com/MozillaFoundation/foundation.mozilla.org/pull/12976/files#r1791075302).